src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration/ConfigurationClientOptions.cs

```diff
@@ -2,28 +2,33 @@
 // Licensed under the MIT License.
 
 using System;
-using Azure.Base.Http;
-using Azure.Base.Http.Pipeline;
+using Azure.Base.Pipeline;
+using Azure.Base.Pipeline.Policies;
 
 namespace Azure.ApplicationModel.Configuration
 {
     public class ConfigurationClientOptions: HttpClientOptions
     {
-        static readonly HttpPipelinePolicy s_defaultRetryPolicy = Base.Http.Pipeline.RetryPolicy.CreateFixed(3, TimeSpan.Zero,
-            //429, // Too Many Requests TODO (pri 2): this needs to throttle based on x-ms-retry-after
-            500, // Internal Server Error
-            503, // Service Unavailable
-            504  // Gateway Timeout
-        );
+        public FixedRetryPolicy RetryPolicy { get; set; }
```

Comment:
Should the type of this be the base `RetryPolicy`?

Reply:
The current thinking is the following consumption model:
```
var options = new ConfigurationClientOptions();
options.RetryPolicy.Delay = ...;
options.RetryPolicy.MaxRetries = ...;

var client = new ConfigurationClient(options);
```

It has its downsides but that's the iteration we're on.